specification/security/resource-manager/Microsoft.Security/stable/2024-08-01/securityStandards.json

```diff
@@ -0,0 +1,391 @@
+            "Default",
+            "Compliance"
+          ],
+          "x-ms-enum": {
+            "name": "standardType",
+            "modelAsString": true
+          }
+        },
+        "description": {
+          "description": "description of the standard",
+          "type": "string"
+        },
+        "assessments": {
+          "description": "List of assessment keys to apply to standard scope.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssessmentProperties"
+          },
+          "x-ms-identifiers": []
+        },
+        "cloudProviders": {
+          "description": "List of all standard supported clouds.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StandardSupportedCloud"
+          }
+        },
+        "policySetDefinitionId": {
+          "description": "The policy set definition id associated with the standard.",
+          "type": "string"
+        },
+        "metadata": {
+          "$ref": "#/definitions/StandardMetadata",
+          "description": "The security standard metadata."
+        }
+      }
+    },
+    "StandardMetadata": {
+      "type": "object",
+      "description": "The standard metadata",
+      "properties": {
+        "createdBy": {
+          "description": "standard Created by object id (GUID)",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdOn": {
+          "description": "standard creation date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "lastUpdatedBy": {
+          "description": "standard last updated by object id (GUID)",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastUpdatedOn": {
+          "description": "standard last update date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "AssessmentProperties": {
+      "type": "object",
+      "description": "Describes properties of an assessment as related to the standard",
+      "properties": {
+        "assessmentKey": {
+          "description": "the assessment key",
+          "type": "string"
+        }
+      }
+    },
+    "StandardSupportedCloud": {
+      "type": "string",
+      "description": "The cloud that the standard is supported on.",
+      "enum": [
+        "Azure",
+        "AWS",
+        "GCP"
+      ],
+      "x-ms-enum": {
+        "name": "StandardSupportedCloud",
+        "modelAsString": true
+      }
+    }
+  },
+  "parameters": {
+    "StandardId": {
+      "name": "standardId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "pattern": "[{]?[0-9a-fA-F]{8}-(?:[0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}[}]?$",
+      "description": "The Security Standard key - unique key for the standard type",
+      "x-ms-parameter-location": "method"
+    },
+    "SecurityStandardBody": {
```

Comment:
>SecurityStandardBody [](http://example.com/codeflow?start=5&length=20)

you can then use just SecurityStandard then ? no need for SecurityStandardBody

Reply:
@ramoka178 
This is the same
When you create the security standard, you give the body as a parameter
in the response, you can see that there are also the resource details (id. name, type and etc...) this is the norma that we are using in Micorosft.security